assets/icons/deprecated/table.svg

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Please don't add or edit any icons in the `deprecated/` directory. That directory is designed to mimic our existing icon font. The only thing we'll ever do here is delete files once we stop using them (and thus no longer need them in the icon font).

Reply:
Ok